packages/@okta/vuepress-site/guides/sign-into-mobile-app/check-for-tokens/android/checkfortoken.md

```diff
@@ -1 +1,56 @@
-check for token on startup code example
\ No newline at end of file
+After `AuthenticateClient` instance is created you can check if the user is logged in with the following:
+
+```java
+if (client.isLoggedIn()) {
+    //user already logged in. Skip login screen.
+}
+```
+
+If the user is logged in and you want to check if the `access_token` is still valid you can check the validity of the token in two ways the first is to simply check the expiration time:
```

Comment:
If the user is signed in and you want to make sure that the `access_token` is still valid, you can check the validity of the token in two ways. The first way is to simply check the expiration time: